index.src.html

```diff
@@ -3998,6 +4030,13 @@
             [=combined buffer layout/allocationSize=].
     9. Return |combinedLayout|.
 
+: <dfn for=VideoFrame>Copy VideoFrame metadata</dfn> (with |metadata|)
+:: 1. Let |metadataCopySerialized| be [$StructuredSerialize$](|metadata|).
```

Comment:
StructuredSerialize/StructuredDeserialize don't seem to linkify in the preview. Do we need some other tagging somewhere to make that happen?

Reply:
It linkifies locally for me when using https://api.csswg.org/bikeshed/.
I am not sure why it does not do this in the preview.

Reply:
The links do work in [the diff preview link](https://pr-preview.s3.amazonaws.com/w3c/webcodecs/559/938df12...youennf:673a0fa.html).

Reply:
Seems to work now, I am not sure why